Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -45,22 +45,34 @@ topCardView config push =
   , padding $ Padding 16 16 16 16
   , gradient $ Linear (if os == "IOS" then 90.0 else 0.0) [Color.black900, Color.black900, Color.pickledBlue, Color.black900]
   , gravity CENTER
-  ][  linearLayout
+  ][  relativeLayout
       [ width MATCH_PARENT
       , height WRAP_CONTENT
-      , gravity RIGHT
-      , layoutGravity "right"
       , padding $ PaddingTop safeMarginTop
-      ][  imageView
-          [ height $ V 40
-          , width $ V 40
-          , accessibility config.accessibility
-          , accessibilityHint "Contact Support : Button"
-          , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
-          , onClick push $ const Support
-          ]
+      ][  
+          linearLayout[
+            width MATCH_PARENT
+          , height WRAP_CONTENT
+          , gravity CENTER 
+          , margin $ MarginTop 5
+          ][if config.topCard.topPill.visible then topPillView config push else dummyTextView]
+        , linearLayout[
+            width MATCH_PARENT
+          , height WRAP_CONTENT
+          , gravity RIGHT
+          ][
+          imageView
+            [ height $ V 40
+            , width $ V 40
+            , accessibility config.accessibility
+            , accessibilityHint "Contact Support : Button"
+            , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
+            , onClick push $ const Support
+            , gravity RIGHT
```

Comment:
@S-PRAVINKUMAR  remove gravity if not required.

Reply:
gravity will not work in imageView